tests/test_response_headers.py

```diff
@@ -0,0 +1,21 @@
+import os
+import pytest
+import threescale_api
+
+
+@pytest.fixture(scope="module")
+def cl():
```

Comment:
There is a fixture already in conftest, see https://github.com/3scale-qe/3scale-api-python/blob/5509cb60a98669f03dcff9f331a233410ce04bf4/tests/integration/conftest.py#L61

Reply:
Also please squash all commits in one PR into one. In this case there is no need to have more than one commit in one PR.